src/controllers/workPeriods.controller.ts

```diff
@@ -3,10 +3,14 @@ import { Request, Response } from 'express';
 import type WorkPeriodsService from '../services/workPeriods.service';
 import type { CreateWorkPeriodRequest, DeleteWorkPeriodRequest, GetWorkPeriodRequest, UpdateWorkPeriodRequest, WorkPeriod } from '../types/workPeriod';
 
+type Config = {
```

Comment:
Config does seem too generic and may create confusion in a larger codebase. Renaming it to ﻿WorkPeriodsContext would provide more clarity and specificity